libs/execution_context/ebpf_program.c

```diff
@@ -2353,7 +2353,7 @@ ebpf_program_execute_test_run(
     test_run_context->program = program;
     test_run_context->program_data = program_data;
     test_run_context->required_irql = program_data->required_irql;
-    test_run_context->context = context;
+    test_run_context->context = NULL;
```

Comment:
Do we still need context in test_run_context here? Seems like it could be local to _ebpf_program_test_run_work_item.